utility/mmsp2txt.cpp

```diff
@@ -0,0 +1,672 @@
+// mmsp2txt.cpp
```

Comment:
This appears to generate exactly the same output as `mmsp2csv`, but with a different file extension. If so, the user should run `mmsp2csv` and simply specify the output filename as the final argument.
Any ASCII TXT utility ought to produce space- or tab-delimited output, more readily legible to the human eye than CSV. `mmsp2tsv` already provides the latter.